crates/matrix-sdk-base/src/client.rs

```diff
@@ -1252,6 +1257,16 @@ impl BaseClient {
         // above. Oh well.
         new_rooms.update_in_memory_caches(&self.store).await;
 
+        for (room_id, member_ids) in updated_members_in_room {
+            if let Some(room) = self.get_room(&room_id) {
+                if let Err(err) =
+                    room.room_member_updates_sender.send(RoomMembersUpdateKind::Partial(member_ids))
+                {
+                    warn!("Failed to broadcast partial room members update: {}", err);
+                }
```

Comment:
Let's not warn anything, since the only error that can happen is that there's no one listening at this point:

```suggestion
                let _ = room.room_member_updates_sender.send(RoomMembersUpdateKind::Partial(member_ids));
```